src/components/home/slider/sliderModal.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react'
+import {FC} from 'react'
+import * as SliderParent from './style'
+import {Icon} from 'react-icons-kit'
+import {ModalProvider} from '../../App'
+
+// icons
```

Comment:
If you like adding these for now I suggest prepending them with `NOTE`. E.g. `// NOTE: icons`

Reply:
Since Iam lately used a package for `Icons` Iam considering deleting them is better.